### DISCUSSION TITLE
plugin help

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23148] [Q&A]
Hi

I want to make a plugin that changes all grass tiles to grass tiles with snow, when it's snowing.
When it stops snowing, they return to normal grass.

Is this possible in a plugin and how to change grass tiles? Where are they , how to edit and save them again. Also how to display them in the plugin?

Thanks

**Comment by telk5093** [comment]
![image](https://github.com/user-attachments/assets/c6ef0c33-8a7c-43f3-bd41-ecdc4becc5a9)
Please don't send email personally. I'm not a personal teacher.
